loader/stage1/stage1.c

```diff
@@ -109,6 +109,7 @@ void stage1_main(evmm_desc_t *xd)
 		print_panic("%s: Failed init efi services\n", __func__);
 		return;
 	}
+	xd->top_of_mem = efi_get_tom();
```

Comment:
check if it is 0. we should allow some stage0 to set it with special method.
in this patch, the "TOP_OF_MEM" should be removed from cfg.

Reply:
0 will be checked in line 117.  if cfg removed, then stage0 need to be changed